pkg/controller/pod_volume_restore_controller.go

```diff
@@ -56,38 +56,46 @@ import (
 
 func NewPodVolumeRestoreReconciler(client client.Client, mgr manager.Manager, kubeClient kubernetes.Interface, dataPathMgr *datapath.Manager,
 	counter *exposer.VgdpCounter, nodeName string, preparingTimeout time.Duration, resourceTimeout time.Duration, podResources corev1api.ResourceRequirements,
-	logger logrus.FieldLogger) *PodVolumeRestoreReconciler {
+	logger logrus.FieldLogger, dataMovePriorityClass string) *PodVolumeRestoreReconciler {
+	log := logger.WithField("controller", "PodVolumeRestore")
```

Comment:
It doesn't look like necessary as we have already printed the same log in Velero server startup or node-agent server startup.